avaframe/com7Regional/com7RegionalCfg.ini

```diff
@@ -0,0 +1,41 @@
+# Configuration settings for regional run
+[GENERAL]
+#-------Input-------
+# use functions from splitInputs.py (in4Region) to split inputs prior to regional run (not in use yet)
+splitInputs = True
+
+#-------Output-------
+# set to 'True' to gather all output peakFiles from the avaDirectories
+# into a single 'allPeakFiles' directory within the regional directory. this also moves all the saved timeSteps.
+movePeakFiles = True
+# set to 'True' to copy the peakFiles and tSteps instead of moving them. only applies if movePeakFiles = True
+copyPeakFiles = False
+
+[com1DFA_com1DFA_override]
+# use default com1DFA config as base configuration (True) and override following parameters
+# if False and local_com1DFACfg is available use local
+defaultConfig = True
+
+# expected mesh size [m]
```

Comment:
Mh if you need this for development purposes, fine, but I would not set a cellsize, but instead work on what's given.